federation/backends/collections.md

```diff
@@ -1,6 +1,40 @@
 # Collections
 
-If back-ends offer/mirror the same datasets, it is required to align the metadata.
+If back-ends offer/mirror the same datasets, it is required to align names and metadata.
+
+The following standardized names have been agreed upon:
+
+- SENTINEL2_L1C
+- SENTINEL2_L2A
+- SENTINEL1_GRD
```

Comment:
Nitpick, I'd prefer a numerical/alphabetic order here. Otherwise it will get a bit confusing with more collections. 

```
- SENTINEL1_GRD
- SENTINEL2_L1C
- SENTINEL2_L2A
- SENTINEL3_OLCI_L1B
```

Reply:
done